modules/standard/MemMove.chpl

```diff
@@ -202,6 +204,7 @@ module MemMove {
                       'more indices fall outside its domain');
   }
 
+  // TODO: this function is unused, can it be removed?
```

Comment:
Maybe ask about these to the team and just rip them out in this PR?

Reply:
I confirmed that these were just left behind by https://github.com/chapel-lang/chapel/pull/21758/ and https://github.com/chapel-lang/chapel/pull/26540 and they can be removed, which I have done